native_client/README.md

```diff
@@ -0,0 +1,48 @@
+# DeepSpeech native client
+
+A native client for running queries on an exported DeepSpeech model.
+
+## Requirements
+
+* [TensorFlow source](https://www.tensorflow.org/install/install_sources)
+* [libsox](https://sourceforge.net/projects/sox/)
+
+## Preparation
+
+Create a symbolic link in the TensorFlow checkout to the deepspeech `native_client` directory.
+
+```
+cd tensorflow
+ln -s ../DeepSpeech/native_client ./
+```
+
+## Building
+
+Before building the TensorFlow stand-alone library, you will need to prepare your environment to configure and build TensorFlow. Follow the [instructions](https://www.tensorflow.org/install/install_sources) on the TensorFlow site for your platform, up to the end of 'Configure the installation'.
+
+To build the TensorFlow library, execute the following command:
+
+```
+bazel build -c opt //tensorflow:libtensorflow.so
```

Comment:
I think you should also mention the configure step or reference the official documentation or both.